data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	"Captain Last! Wake up!" Her cheerful demeanor is a grating, but effective alarm clock.`
+			label goto7
+			`	"Captain <last>, the proceedings have completed, and your time here is at an end. The Elder Council voted not to punish Teeneep, probably because she is about to trigger an election. Some Elders have referred to her as Eyrisen of the Mythos," she giggles, "and if I was religious, I'd agree.`
+			`	"I'll escort you to Teeneep's and Sayari's waiting room now."`
+				goto return7
+
+			label library
+			choice
+				`	INT-8Y-84X19H4: Solutions to Impending Problems`
+					goto solutions
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: solutions book"
+				`	HUM-9U4C12A: Troubles in Human Space`
+					goto human
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: human book"
+				`	HUM-9XC401Q: The Pirate Problem`
+					goto pirates
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: pirates book"
+				`	UHI-001AX94: Unfettered Military Activity and its Economic Impact on the Hai`
+					goto unfettered
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: unfettered book"
+				`	UHI-00201A3: Unfettered Food Situation`
+					goto acorns
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: acorns book"
+				`	CHS-015TNP3: A Lifetime of Disruption: Teeneep Before Public Life`
+					goto teeneep1
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: teeneep1 book"
+				`	CHS-015TNP6: Chaos Personified: Teeneep's Rise to Fame and Infamy`
+					goto teeneep2
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: teeneep2 book"
+				`	QRG-3Z-18FJ4KC: Modern Hai-Quarg Relations`
+					goto quarg
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: quarg book"
+				`	HUM-A0C1I3X: Comparisons of Hai and Human Military Technology`
+					goto militech
+					to display
+						not "Hai Reveal [A11] Sound the Alarm: militech book"
+				`	"I've changed my mind. Take me back to the proceedings instead."`
+					goto reading
+					to display
+						"Hai Reveal [A11] Sound the Alarm: time" == 2
+				`	"I'm taking a nap. Let me know when it's all over."`
+					goto nap
+					to display
+						"Hai Reveal [A11] Sound the Alarm: time" > 2
+
+			label solutions
+			action
+				set "Hai Reveal [A11] Sound the Alarm: solutions book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+				log `A Hai intelligence report identifies three problems: "northern troubles," degrading secrecy of the Hai, and piracy. They expect piracy to vanish from the Republic "befriending" anarchist colonies and suggest Hai slowly reveal themselves through leaked information. It is unclear if that ever happened. For the "northern troubles," they have a "skilled mercenary" in mind for the job, and a diplomat.`
+			`	This one-page report is recent, and includes references to other reports you can access.`
```

Comment:
I think that we should get a mention of it being stored in the logs?
Or is the idea FL just remembers about this? Or logs this like all other logs?

Reply:
However the other logs are stored is how these are stored.

Reply:
Fair. I was just pointong out you could somehow tell it to the player
"You took good notes of the reportw in your logs" when ur done reading for instance